src/patternfly/utilities/variables.scss

```diff
@@ -99,6 +102,7 @@ $fa-font-path: "./assets/fonts/webfonts" !default;
 $pf-global--image-path: "/assets/images" !default;
 
 // Spacers
+$pf-global--spacer--xxs: pf-size-prem(2px) !default;    // Color in the visuals
```

Comment:
If we need to add more spacers on a global level lets first check with the designers. @mceledonia